src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
If we do keep the method, it would be better to use the word Execute rather than Run for consistency. And add some Javadoc to explain it more fully. I'm not getting enough from the example name.